app/uk/gov/hmrc/exports/models/declaration/submissions/Action.scala

```diff
@@ -38,10 +91,50 @@ object Action {
   implicit val readLocalDateTimeFromString: Reads[ZonedDateTime] = implicitly[Reads[LocalDateTime]]
     .map(ZonedDateTime.of(_, ZoneId.of("UTC")))
 
-  implicit val writes = Json.writes[Action]
+  private val allWrites = (JsPath \ "id").write[String] and
```

Comment:
All these different read/writes are another downside of the Action trait approach